### DISCUSSION TITLE
Scoopbox UI tool

### DISCUSSION THREAD

**Discussion by LAB02-Admin** [hasan-hasanov/ScoopBox#5] [Show and tell]
Hi @hasan-hasanov,

Not really a discussion, just wanted to let you know I created a small UI tool to manage your great library:

https://github.com/LAB02-Research/ScoopBoxManager

It's usable, only custom scripts aren't implemented yet, but that'll follow soon.

Thanks for sharing your work, really happy with it!

**Comment by hasan-hasanov** [comment]
Looks very cool. Keep up the great work. I allowed myself to list you in the ScoopBox readme. Have a great day.

**Comment by LAB02-Admin** [comment]
Glad you like it, thanks for listing!

> Have a great day

You as well :)
